Resources/Private/Scss/_main.scss

```diff
@@ -13,15 +11,15 @@
 
 .doubleColumn {
     @include media-l {
-        @include width-l-12;
+        width: width(l-12);
     }
```

Comment:
I'm still confused about the way media queries are used (here and elsewhere). why not "width(12)" and take care about media query related things within the function/mixin. makes everything cleaner and less error-prone.